virtualworkspace/provider.go

```diff
@@ -34,12 +34,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/cluster"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+	mcmanager "sigs.k8s.io/multicluster-runtime/pkg/manager"
+	"sigs.k8s.io/multicluster-runtime/pkg/multicluster"
```

Comment:
Have you updated gci in golangci-lint?

Reply:
Should we? Since this is outside multicluster-runtime, i would treat this as any other dependency (such as controller-runtime).

Reply:
Ack, it's about taste. Was just wondering why we had it before in a special group. Just because it was not ordered or because we have a group github.com/multicluster-runtime ?

Reply:
We do have a group, so I've adjusted the config. Let's keep that in, I don't want to change that too much (I didn't realise we have the group).